runtime/tests/integration_tests.rs

```diff
@@ -127,6 +134,39 @@ fn test_fw_info() {
     assert_eq!(info.pl0_pauser, 0xFFFF0000);
 }
 
+#[test]
+fn test_disable_attestation_cmd() {
+    let mut model = run_rt_test(None);
+
+    let payload = MailboxReqHeader {
+        chksum: caliptra_common::checksum::calc_checksum(
+            u32::from(CommandId::DISABLE_ATTESTATION),
+            &[],
+        ),
+    };
+
+    let resp = model
```

Comment:
nit: Can you add a comment that once DPE APIs are enabled, we should make sure the RT alias key in the cert is different from the key that signs DPE keys (i.e. make sure the signature check fails)